etc/check_ipmi

```diff
@@ -27,6 +28,13 @@ if eeprom_key == 'ignore':
 try:
     allow_16bit = eeprom_key == '16-bit'
     image = eeprom.read_eeprom(allow_16bit)
+except OSError as e:
+    if e.errno == errno.ENXIO:  # No such device or address (i.e. no FMC EEPROM detected)
+        print('FMC was expected but none detected')
+        sys.exit(1)
+    else:
+        print('Unable to read FMC EEPROM:', e)
```

Comment:
Oh, that's irritating.  Just noticed that lines 36-37 repeat lines 38-39.  I really can't think of a decent workaround, so meh.